### PR TITLE
Fix site and request owner look up

### PR DESCRIPTION
**Pull request by jdorweiler**
#18 added subdomains when parsing URLs. We need to update the owner look up for the site and requests to work with subdomains.

### CODE REVIEW

File: src/trackers/classes/site.js [hunk 1]
```diff
@@ -54,10 +55,8 @@ function _getCookies (siteData) {
  * @param {string} url - url to test against.
  * @returns {bool} True if the url is in this sites first party set.
  */
-function _isFirstParty(url) {
-    const data = new ParsedUrl(url)
```
Comment (jdorweiler) [inline src/trackers/classes/site.js:58]: Don't need this. The Request already has the parsed url in it.

File: package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "eslint": "^6.8.0"
   },
   "scripts": {
-    "build": "./src/trackers/process-crawl.js && ./src/trackers/build-trackers.js",
+    "build": "NODE_OPTIONS=--max-old-space-size=8096 ./src/trackers/process-crawl.js && ./src/trackers/build-trackers.js",
```
Comment (kdzwinel) [inline package.json:33]: TIL! `NODE_OPTIONS` magically passes everything to node? (we should probably prefix all TR scripts with that 😆)

File: src/trackers/helpers/getOwner.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const sharedData = require('./sharedData.js')
+
+function getOwner (domain) {
+    if (!domain) {
+        return
+    }
+    
+    if (sharedData.entityMap.get(domain)) {
```
Comment (kdzwinel) [inline src/trackers/helpers/getOwner.js]: Getting it twice is a bit weird. Maybe let's use `has`?

```suggestion
    if (sharedData.entityMap.has(domain)) {
```

Or maybe

```suggestion
   const owner = sharedData.entityMap.get(domain);
   
   if (owner) {
      return owner;
   }
```

**Review by kdzwinel — COMMENTED** [review]
Nice work abstracting it to a helper! 👏
I found one nit. Feel free to merge once addressed.

**Pull request merged**